src/manifest.jl

```diff
@@ -219,7 +219,11 @@ function read_manifest(f_or_io::Union{String, IO})
         rethrow()
     end
     if Base.is_v1_format_manifest(raw)
-        raw = convert_v1_format_manifest(raw)
+        if isempty(raw) # treat an empty Manifest file as v2 format for convenience
```

Comment:
If someone has a newline or something in there for some reason (maybe they did `echo "" > Manifest-v1.11.toml`), seems like we can still treat that as v2 format?

```suggestion
        if isempty(strip(raw)) # treat an empty Manifest file as v2 format for convenience
```

Reply:
Oh wait, I completely misread this code.

Reply:
`raw` is already a dict, so this is irrelevant.